XenModel/XenModel.csproj

```diff
@@ -353,6 +354,9 @@
     <Compile Include="XenAPI\pgpu_dom0_access.cs" />
     <Compile Include="XenAPI\pool_allowed_operations.cs" />
     <Compile Include="XenAPI\primary_address_type.cs" />
+    <Compile Include="XenAPI\PVS_farm.cs" />
+    <Compile Include="XenAPI\PVS_proxy.cs" />
+    <Compile Include="XenAPI\PVS_server.cs" />

```

Comment:
Because in the previous commit I only pulled the binsings from the SDKs and therefore only added the new files to the repository, without including them in the project.